src/menu_events.cpp

```diff
@@ -454,6 +454,29 @@ void menu_handler::show_enemy_moves(bool ignore_units, int side_num)
 			gui_->highlight_another_reach(path);
 		}
 	}
+	
+	//   Fetch cursor position, in order to later find a possible
+	// unit under it.
+	int x = -1;
+	int y = -1;
+	// auto bitmask =
+			SDL_GetMouseState(&x, &y);
+
+	//   Find possible unit (no matter whether friend or foe) under
+	// the cursor.
+	const mouse_handler& mh = const_cast<mouse_handler&>(
+			pc_.get_mouse_handler_base());
```

Comment:
You should not use `const_cast` here. Just use a non-const reference and remove the similar `const_cast` below. There's also no need for this line to be split. Please make it one line.

Reply:
The reason for it was [coding standards](https://wiki.wesnoth.org/CodingStandards) allows long lines but does not recommend them. Will comply request.